### PR TITLE
PLAT-10817: documented how to solve self signed certificate issues

### PR DESCRIPTION
**Pull request by symphony-elias**
### Ticket
PLAT-10817

### Description
Documented how to solve SSL issues

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [N/A] Unit tests updated or added
- [N/A] Docstrings added or updated
- [x] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: docsrc/markdown/known_issues.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Known issues
```
Comment (symphony-youri) [inline docsrc/markdown/known_issues.md]: I would not call it an issue but a workaround 😛

File: docsrc/markdown/known_issues.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+If you use different hosts and ports for the `agent`, `keyManager` and `sessionAuth`, you will have to repeat the
+command above with the specific hosts and ports and append all the certificates to the same file.
+
+Then, add the following to the configuration:
+
+```yaml
+ssl:
+  trustStore:
```
Comment (symphony-youri) [inline docsrc/markdown/known_issues.md]: Would adding the certificate to the system keychain be a workaround too?

File: docsrc/markdown/known_issues.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/sslproto.py", line 189, in feed_ssldata
+self._sslobj.do_handshake()
+
+File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ssl.py", line 944, in do_handshake
+self._sslobj.do_handshake()
+
+ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1123)
+```
+
+means your pod uses a self-signed certificate.
```
Comment (symphony-youri) [inline docsrc/markdown/known_issues.md]: Does it mean that develop2 is a self signed certificate? It appears ok here: https://www.sslshopper.com/ssl-checker.html#hostname=https://develop2.symphony.com

Comment (symphony-elias) [reply docsrc/markdown/known_issues.md]: This is strange indeed... I will update the sentence.

File: docsrc/markdown/known_issues.md (hunk 1 above)
Comment (symphony-elias) [reply docsrc/markdown/known_issues.md]: Should FAQ be a better name ?

File: docsrc/markdown/known_issues.md (hunk 3 above)
Comment (symphony-youri) [reply docsrc/markdown/known_issues.md]: Actually with https://www.ssllabs.com/ssltest/analyze.html?d=develop2.symphony.com&latest the last digicert in the chain is indeed marked as self signed

**Review by symphony-youri — APPROVED** [review]
An alternative would be to support an insecure flag (similar to curl -k) but I'm not a big fan of supporting this

**Pull request merged**